lib/ledger.js

```diff
@@ -388,9 +396,23 @@ async function _sendElectorPool({electorPoolId, maintainerDoc}) {
 
   // pick one node to send the electorPool document to
   const [target] = targets;
-  const operation =
-    await target.client.wrap({record: electorPoolDoc});
-
+  // gets the ledger id from the target
+  const getLedgerId = ({client = {}} = {}) => {
+    const {
+      ledgerAgentStatus: {
+        latestConfigEvent: {
+          ledgerConfiguration: {
+            ledger
+          } = {}
+        } = {}
+      } = {}
+    } = client;
+    if(ledger) {
+      return ledger;
+    }
+    throw new Error('Unable to determine ledger for current target');
+  };
```

Comment:
This is unnecessarily complex. The function that calls `_sendElectorPool` can just pass in the ledger ID. I see it's called from the function that sets it here:

https://github.com/veres-one/veres-one/blob/4d17de37351a3f067d74dc016337b420f9563659/lib/ledger.js#L188

So just add a `ledgerId` param to `_sendElectorPool` and pass it in.

```suggestion
```